.circleci/config.yml

```diff
@@ -443,6 +443,14 @@ jobs:
       - sfchangecase/update:
           location: REPO_LOCATION
           release: GUS_RELEASE
+      - slack/notify:
```

Comment:
I might go ahead and remove the post merge steps altogether (depending on the results of the automation in the next few weeks) - instead having a reminder posted to slack to perform the post merge. This might just not be worthwhile spending much more time on to have circle ci automate for us.

Reply:
Sounds good, the post merge is a difficult thing to automate but shouldn't be an issue to keep it as a manual step